rabbitmq/nginx_wo_ssl/nginx_wo_ssl.conf

```diff
@@ -0,0 +1,15 @@
+upstream smm_smile_flask {
+    server web:8001;
```

Comment:
do you mind i change this line to `smile-server:8001`? i was copying from anther project where `web` is that server's container name

Reply:
smile-server is not the correct answer either. it should be actual server name

Reply:
i should change this to `smm-tester.ncsa.illinois.edu:8001` ?